src/solver/type_getters.jl

```diff
@@ -517,6 +517,18 @@ function get_callbacks(parsed_args, sim_info, atmos, params, comms_ctx)
     return callbacks
 end
 
+function get_output_dir(config::AtmosConfig)
+    (; parsed_args) = config
+    job_id = if isnothing(parsed_args["job_id"])
+        job_id_from_config(parsed_args)
+    else
+        parsed_args["job_id"]
+    end
```

Comment:
Let's not duplicate logic to get the job ID

Reply:
I moved it to a separate function.